File: src/main/resources/output/IFC4X3_js/Entities/IfcPresentationItem/IfcSurfaceTexture/IfcBlobTexture/IfcBlobTexture.js
class IfcBlobTexture extends IfcSurfaceTexture {
    constructor() {
        /** @type {IFCIDENTIFIER} */
        this.RasterFormat = null;
        /** @type {IFCBINARY} */
        this.RasterCode = null;
    }

    // === WHERE CLAUSES ===
    // RasterCodeByteStream : BLENGTH(RasterCode) MOD 8 = 0
    // SupportedRasterFormat : SELF.RasterFormat IN ['BMP', 'JPG', 'GIF', 'PNG']
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcPresentationItem/IfcSurfaceTexture/IfcBlobTexture/IfcBlobTexture.py
class IfcBlobTexture(IfcSurfaceTexture):
    def __init__(self):
        self.RasterFormat: IfcIdentifier = None
        self.RasterCode: IfcBinary = None

    # === WHERE CLAUSES ===
    # RasterCodeByteStream : BLENGTH(RasterCode) MOD 8 = 0
    # SupportedRasterFormat : SELF.RasterFormat IN ['BMP', 'JPG', 'GIF', 'PNG']
